### PR TITLE
add gbd 2019 hhtb model

### PR DESCRIPTION
**Pull request by yongqx**
This household TB model is updating to use GBD 2019 data. The prevalence of household contact is calculated by using:
1 - Pr[no active TB in house] among those who susceptible to active TB for certain age, sex, and location.

### CODE REVIEW

File: src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import pandas as pd
+from get_draws.api import get_draws
+from vivarium.interpolation import Interpolation
+
+master_dir = '/home/j/Project/simulation_science/latent_tuberculosis_infection/literature/household_structure/microdata/'
```
Comment (collijk) [inline src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py:6]: We don't want file paths in our public facing repos.

Comment (aflaxman) [reply src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py:6]: I've seen them in many other repos, e.g. https://github.com/ihmeuw/vivarium_csu_swissre_breast_cancer/blob/master/src/vivarium_csu_swissre_breast_cancer/paths.py#L7; why don't we want them?

Comment (yongqx) [reply src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py:6]: > We don't want file paths in our public facing repos.

Do we have `global_path` for the household microdata?

File: src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+def get_estimates(location_id: int, draw: int):
+    """calculate the prevalence of household contact"""
+    cols = ['age_group_start', 'age_group_end', 'sex']
+    df_hh = load_hh_data(location_id)
+    hh_ids = df_hh.hh_id.unique()
+    # boostrap HH data by resampling hh_id with replacement
+    sample_hhids = np.random.choice(hh_ids, size=len(hh_ids), replace=True)
+    df_hh = df_hh.set_index("hh_id")
+    df_hh_sample = df_hh.loc[sample_hhids].reset_index()
+    prev_actb = format_actb_prevalence(pull_actb_prevalence(location_id))
```
Comment (yongqx) [inline src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py]: `line 137` slow things down, we may want to update the ltbi artifact for active TB prevalence in order to pull GBD 2019 data. @collijk Our artifact seems promising for getting single draw data, right?

File: src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            result = pd.concat([result, age_sex_result], ignore_index=True)
+    return result
+
+def get_estimates(location_id: int, draw: int):
+    """calculate the prevalence of household contact"""
+    cols = ['age_group_start', 'age_group_end', 'sex']
+    df_hh = load_hh_data(location_id)
+    hh_ids = df_hh.hh_id.unique()
+    # boostrap HH data by resampling hh_id with replacement
+    sample_hhids = np.random.choice(hh_ids, size=len(hh_ids), replace=True)
```
Comment (yongqx) [inline src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py]: @aflaxman For `line 134`, if we change `size=min(len(hh_ids), max_hh_sample)`. What number would be appropriate for `max_hh_sample`? Can we use the mean/median of the hh_id number across 20 countries? alternatively, we could set it to be X% (e.g., 25%) of the largest hh_id number.

Comment (aflaxman) [reply src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py]: What is the mean, max and 95-th percentile of these values?  What is the largest number you are willing to wait for?

Comment (yongqx) [reply src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py]: Summary of `hh_id_count` across 20 countries

mean   4.283952e+05
std    9.959896e+05
min    7.048000e+03
25%    1.438475e+04
50%    1.697500e+04
75%    4.230725e+04
max    3.600005e+06

**Comment by aflaxman** [comment]
Max of 50,000 seems safe and fast to me, then.

**Pull request merged**